python/kserve/kserve/handlers/http.py

```diff
@@ -65,20 +64,63 @@ def get_model(self, name: str):
 
 
 class PredictHandler(HTTPHandler):
+    def create_cloudevent(self, model_name: str, body: Union[Dict, CloudEvent], response: Dict,
```

Comment:
Can we move this function to utils? suggest renaming to `create_response_cloudevent` as it is specific to response.